config/auth.php

```diff
@@ -116,4 +116,18 @@
     */
 
     'password_timeout' => 10800,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Email Verification Timeout
+    |--------------------------------------------------------------------------
+    |
+    | Here you may define the amount of minutes before an email verification
+    | times out and the user is prompted to re-verify their email via the
+    | verify screen. By default, the timeout lasts for one day.
+    |
+    */
+    'verification' => [
+        'expire' => 1440,
+    ],
```

Comment:
Where is this used? Couldn't find any documentation about it.

Reply:
https://github.com/laravel/framework/blob/7907bfb50165b3951403e2d9882b4f8e0ee26b9e/src/Illuminate/Auth/Notifications/VerifyEmail.php#L85

Not documented for included in default laravel auth config